sdk/tests/platform/test_az_mqtt5_policy.c

```diff
@@ -284,11 +290,44 @@ static void test_az_mqtt5_policy_outbound_sub_success(void** state)
   assert_int_equal(ref_suback, 1);
 }
 
+static void test_az_mqtt5_policy_outbound_pub_no_properties_success(void** state)
+{
```

Comment:
If we're requiring an empty property bag on a pub without properties, should we also be requiring an empty property bag on a sub with no properties? Internally, we also have
> sub_data->properties ? sub_data->properties->properties : NULL));

Reply:
Changed implementation to not require a property bag if it will not be used.